plugin/builtin/imagetagtransformer/go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	sigs.k8s.io/kustomize/api v0.3.1
 	sigs.k8s.io/yaml v1.1.0
 )
+
+replace sigs.k8s.io/kustomize/api => ../../../api
```

Comment:
Is this necesssary?

Reply:
Yes, otherwise the test will not use the local repository version with the updated change.

Reply:
IIUC this reverts the work of 

https://github.com/kubernetes-sigs/kustomize/commit/b8a13b633577dbcba2dcadcf0e80f6586ccd64ad#diff-70da81bc5868bb9587cc9483698931bc

Should we do this as 3 steps -- update the api, release it, and then update the plugin?

@monopole how is this normally done?

Reply:
Let's release a new version if kusotmize/api. Then update the plugin to depend on the new api version.

Reply:
If I understand correctly, the `api/builtins/ImageTagTransformer.go` change should be a separate pull request?

Reply:
Yes.

Reply:
Sorry I was busy with some other tasks. Added a separate PR that only includes the API change: https://github.com/kubernetes-sigs/kustomize/pull/2249